online/CMakeLists.txt

```diff
@@ -0,0 +1,70 @@
+ ################################################################################
+ #    Copyright (C) 2014 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH    #
```

Comment:
```suggestion
 #    Copyright (C) 2022 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH    #
```

Reply:
Thanks. I completely forgot about it. I will update also the one in base.

Reply:
Done.